src/object/srv_obj.c

```diff
@@ -1994,6 +1994,10 @@ static int
 obj_capa_check(struct ds_cont_hdl *coh, bool is_write, bool is_agg_migrate)
 {
 	if (!is_write && !ds_sec_cont_can_read_data(coh->sch_sec_capas)) {
+		if (uuid_compare(coh->sch_cont->sc_pool->spc_pool->sp_srv_cont_hdl,
+				 coh->sch_uuid) == 0)
```

Comment:
seems for srv_cont_hdl, "!ds_sec_cont_can_read_data(coh->sch_sec_capas)" should not be true, not very sure if there is other hidden issue .